src/components/RecordTypesSelect/RecordTypesSelect.js

```diff
@@ -24,24 +31,29 @@ const useUpdateOnResize = () => {
   }, [forceUpdate]);
 };
 
-const incomingRecord = FOLIO_RECORD_TYPES.MARC_BIBLIOGRAPHIC;
-
-export const RecordTypesSelect = ({
+export const RecordTypesSelect = memo(({
   id,
   existingRecordType,
-  onRecordSelect,
+  incomingRecordType,
+  onExistingSelect,
+  onIncomingSelect,
   isEditable,
 }) => {
   useUpdateOnResize();
   const [existingRecord, setExistingRecord] = useState(undefined);
+  const [incomingRecord, setIncomingRecord] = useState(undefined);
 
   useEffect(() => {
     setExistingRecord(FOLIO_RECORD_TYPES[existingRecordType] || undefined);
   }, [existingRecordType]);
 
+  useEffect(() => {
+    setIncomingRecord(INCOMING_RECORD_TYPES[incomingRecordType] || undefined);
```

Comment:
I think you can rewrite it to `setIncomingRecord(INCOMING_RECORD_TYPES?.[incomingRecordType])` and it must works. This does not have to be changed, it is just a suggestion.

Reply:
Thanks, @Vladyslav-Velytskyi! Rewrote.